src/BillSplit.test.tsx

```diff
@@ -1,10 +1,9 @@
 import Typography from "@material-ui/core/Typography/Typography";
```

Comment:
This became an unused import because you removed its use on line 8 (and thus CI failed).  Fixed